install/kubernetes/helm/subcharts/prometheus/templates/configmap.yaml

```diff
@@ -11,13 +11,10 @@ metadata:
 data:
   prometheus.yml: |-
     global:
-      scrape_interval: 15s
+      scrape_interval: {{ .Values.scrapeInterval }}
     scrape_configs:
 
     - job_name: 'istio-mesh'
-      # Override the global default and scrape targets from this job every 5 seconds.
-      scrape_interval: 5s
```

Comment:
@douglas-reid One question is removing those override parameters also increase the scrape time for those components, do you still remember why use override value here?

Reply:
It does increase the scrape time, but I don't think there was any particular reason for them other than making it easy to see results in Prometheus sooner for demo purposes, etc. We *may* have also wanted quicker times for e2e tests (for validation). In the end, I'm not sure they are/were needed for any real operational purpose.

Reply:
Thanks @douglas-reid , make sense.